content/en/docs/concepts/workloads/controllers/cron-jobs.md

```diff
@@ -82,6 +80,14 @@ be down for the same period as the previous example (`08:29:00` to `10:21:00`,)
 The CronJob is only responsible for creating Jobs that match its schedule, and
 the Job in turn is responsible for the management of the Pods it represents.
 
+## New controller (alpha)
+
+There's an alternative implementation of the CronJob controller exposed as an alpha feature since kubernetes 1.20. To enable it pass the following feature gate flag to {{< glossary_tooltip term_id="kube-controller-manager" text="kube-controller-manager" >}}.
```

Comment:
```suggestion
There's an alternative implementation of the CronJob controller, available as an alpha feature since Kubernetes 1.20. To select version 2 of the CronJob controller, pass the following [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) flag to the {{< glossary_tooltip term_id="kube-controller-manager" text="kube-controller-manager" >}}.
```

Reply:
Updated.